svgpathtools/svg_to_paths.py

```diff
@@ -1,3 +1,6 @@
+#
+# Modified by NXP 2024, 2025
```

Comment:
Did you leave this attribution in by accident or is there some reason you really want it?

Reply:
hi @mathandy 
I left an explanation here: https://github.com/mathandy/svgpathtools/pull/236#discussion_r2092810815

Is this acceptable to stay in the code? It just says that we, NXP, modified some code on the offending file. It is NXP policy to enforce this upstream.

Reply:
OK, we moved our modified copyright on the if block.
Thanks!